src/Sinch/Conversation/Conversations/Conversations.cs

```diff
@@ -244,20 +245,19 @@ public Task Stop(string conversationId, CancellationToken cancellationToken = de
         }
 
         /// <inheritdoc />
-        public Task<InjectEventResponse> InjectEvent(InjectEventRequest injectEventRequest,
+        public Task<InjectEventResponse> InjectEvent(string conversationId, InjectEventRequest injectEventRequest,
```

Comment:
Why do you need to extract this parameter from `InjectEventRequest` ?

Reply:
The behavior is counterintuitive, If field is marked `required` and `JsonIgnore` at the same type this throws a runtime error from json serializer. I'll include `conversationId` in request class. Also, there will be task to harmonize approach inside all over the solution

Reply:
For the curious - https://github.com/dotnet/runtime/issues/82879